lib/Doctrine/DBAL/Driver/PDOOracle/Driver.php

```diff
@@ -10,8 +10,7 @@
 /**
  * PDO Oracle driver.
  *
- * WARNING: This driver gives us segfaults in our testsuites on CLOB and other
- * stuff. PDO Oracle is not maintained by Oracle or anyone in the PHP community,
+ * WARNING: PDO Oracle is not maintained by Oracle or anyone in the PHP community,
```

Comment:
That statement seems wrong too. This driver seems to be maintained directly in php-src, and the last commit was in October this year: https://github.com/php/php-src/blob/master/ext/pdo_oci/oci_driver.c

Reply:
done